### DISCUSSION TITLE
zie/ziewin/3.0/ja/doc/qb/abtpkg

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#72] [Q&A]
# zie/ziewin/3.0/ja/doc/qb/abtpkg

Z and I Emulator for Windows Version 3.0 パッケージには HCL Z and I Emulator for Windows 、 Version 3.0 が収められており、これにより、5250、3270、および VT エミュレーションと、 z/OS ™ 、 z/VM ™ 、 eServer ™ i5、 System i5 ™ 、 iSeries ™ 、 ...

https://help.hcl-software.com/zie/ziewin/3.0/ja/doc/qb/abtpkg.html

**Comment by KunioMatsuo** [comment]
Add Package details.  Users will be puzzled multiple zip files without any explanation.  I recommend add readme.txt in HCL ZIE For Windows v3.0.zip and this product documentation.
--------------------
2024/08/28  10:16       232,344,785 HCL ZIE For Windows v3.0.zip
contains following 4 zips but no explanation provided.  Users will be puzzled which one is good for me.
2024/04/26  23:26        48,718,298 mchs0423.zip
2024/04/26  23:26        49,315,555 mcht0423.zip
2024/04/27  00:07        80,329,945 mmls0423.zip
2024/04/20  21:04        53,980,529 ZIEWin_3.0_Docs_Admin_Aids.zip

**Reply by KunioMatsuo** [reply]
"mchs" seems Chinese Simplified.  "mcht" seems Chinese Taiwanese.  mmls seems Multi-Lingual Support.

**Comment by KunioMatsuo** [comment]
Neither is not included in installation image.  Those should be removed.
-----
In addition to the products previously mentioned, the Z and I Emulator for Windows installation image contains the following: 
- Adobe Acrobat Reader, to enable you to read softcopy books available on the installation image"
- Book files in PDF format
